avocado_vt/plugins/vt_joblock.py

```diff
@@ -104,7 +104,10 @@ def _lock(self, job):
                 msg = 'File "%s" acquired by PID %u. ' % (path, lock_pid)
                 raise OtherProcessHoldsLockError(msg)
             else:
-                os.unlink(path)
+                try:
+                    os.unlink(path)
+                except OSError:
+                    self.log.warn("Unable to remove stalled lock: %s" % path)
```

Comment:
OK, sounds adequate.  Two little issues: should read "stale" (stalled has a different meaning), and should use arguments after the format string, that is:

```
self.log.warn("Unable to remove stale lock file: %s", path)
```